scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/controllers/ClusterController.java

```diff
@@ -77,6 +79,18 @@ public ClusterSchedulerInfoResponse schedulerInfo() {
         return response;
     }
 
+    @RequestMapping(value = "/scale", method = RequestMethod.POST)
+    public void scaleCluster(@RequestParam("to") String scaleTo) {
```

Comment:
Spring understands types, so should be something like `@RequestParam(…) int sclaeTo`. Then check that number is positive or throw IllegalArgumentException.